typings/globals.d.ts

```diff
@@ -0,0 +1,29 @@
+type Struct = NodeJS.Dict<any>;
```

Comment:
This file is not exported from the package and will, therefore, be ignored by TS.
PTAL https://github.com/metarhia/common/pull/344/files.

Reply:
Try to use `tsc` from cloned repo in terminal